packages/widgets/src/components/TradingScoreboard/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ITradingInfoCardProps } from '@components/TradingInfoCard/types';
+import { SerializedStyles } from '@emotion/react';
+
+export interface ITradingScoreboardProps {
+  itemsPerRow: number;
+  items: Array<ITradingInfoCardProps>;
```

Comment:
I think we should not pass all the props from the `ITradingInfoCardProps` interface here. 
`title, value, unit, icon` should be enough. You can use `Pick` to get the right type.

The same applies to the `onClick` handler and the `renderCard` below.

Reply:
Done

Reply:
The issue still exists (noted in a separate comment).

Reply:
Done